.github/containers/ruby/repo/packages/py-llnl-sina/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
```

Comment:
I am not sure we can have these lines in our repository. These refers to the spack license and not ours.

Reply:
changed to AMS license